lambdas/search/index.py

```diff
@@ -50,6 +73,26 @@ def lambda_handler(request):
         }
         size = 0
         _source = []
+    elif action == 'images':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": IMG_EXTS
+                }
+            }
+        }
+        size = 10
+        _source = []
+    elif action == 'other':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": OTHER_EXTS
```

Comment:
close but not quite what we want. we really should have a bias towards `readme.*` files

Reply:
i guess in the general case that implies a full scan; can we at least look for `ROOT/readme.*`?

Reply:
A simple search for "README" works pretty well to grab just READMEs. Do we need anything fancier than that?